publish_unit_test_results.py

```diff
@@ -804,7 +804,7 @@ def main(token: str, event: dict, repo: str, commit: str, files_glob: str,
 def get_commit_sha(event: dict, event_name: str):
     logger.debug("action triggered by '{}' event".format(event_name))
 
-    if event_name == 'push':
+    if event_name == 'push' or event_name == 'workflow_dispatch':
```

Comment:
Can you do it like in the `elif` below, with `event_name in [...]`? Just to align style of both conditions.

Reply:
Sure, why I missed that 🤦

Reply:
Done